constants.go

```diff
@@ -57,4 +57,10 @@ const (
 
 	// ConnectorOIDC means connector type OIDC
 	ConnectorOIDC = "oidc"
+
+	// AWSInstanceIDLabel is label specifying instance id
+	AWSInstanceIDLabel = "aws/instance-id"
```

Comment:
Such comments do not add any information (basically it repeats the variable name, spelled differently). Where is this constant used and what for? What does it mean: "label specifying instance id"? Any pre-existent AWS knowledge fails to explain the meaning of it (labels on AWS do not uniquely identify anything)

Edit; ah, after reading more code I think I understand. It's a built-in teleport label name. In this case, lets name it `instance-id` (because we can re-use it for other clouds in the future). And yeah, the comment failed to tell me that it's a label name.  :)